docs/product/sentry-basics/concepts/tracing/trace-view.mdx

```diff
@@ -4,47 +4,124 @@ sidebar_order: 30
+
+On the left side is a list of operations, and on the right is their duration and any events, such as errors, which may have occurred in that timeframe.
+
+By default, the waterfall view shows a list of transactions. If you need more granular data and want to see spans as well, click the + button on the left and the waterfall will update and show both. The waterfall view helps surface any performance issues or profiles collected during a given timeframe.
+
+<br />
+
+<div
+  style={{
+    height: "0px",
+    paddingBottom: "calc(66.66% + 41px)",
+    position: "relative",
+    width: "100%",
+  }}
+>
+  <iframe
+    src="https://demo.arcade.software/25H9paIDvBOQp5T3pFeW?embed&show_copy_link=true"
+    title="Trace Details — Performance — sentry — Sentry"
+    frameborder="0"
+    loading="lazy"
+    webkitallowfullscreen
+    mozallowfullscreen
+    allowfullscreen
+    allow="clipboard-write"
+    style={{
+      colorScheme: "light",
+      height: "100%",
+      left: "0px",
+      position: "absolute",
+      top: "0px",
+      width: "100%",
+    }}
+    title="Trace view -> Error"
+  ></iframe>
+</div>
+
+<br />
+### Helpful Tips
+
+Because debugging often involves multiple people or teams, we've made it easy to draw attention to specific areas of the trace and share a link that shows what you've highlighted with your colleagues. To do this, click on the row you'd like to draw attention to and share the resulting URL. Your colleague will see exactly what you want them to.
+
+Whatever you highlight will also be saved if you navigate away from the page and will still be there when you use your browser's back and forward buttons.
+
+If you're doing a comparison and want an easy way to go back and forth between highlighted areas of interest in your trace, you can pin the tabs. When a tab is pinned, the view will be persistent throughout your session.
+
+<br />
+<div
+  style={{
+    height: "0px",
+    paddingBottom: "calc(66.66% + 41px)",
+    position: "relative",
+    width: "100%",
+  }}
+>
+  <iframe
+    src="https://demo.arcade.software/yP3PFytK0ckfetjJTe52?embed&show_copy_link=true"
+    title="Sentry"
+    frameborder="0"
+    loading="lazy"
+    webkitallowfullscreen
+    mozallowfullscreen
+    allowfullscreen
+    allow="clipboard-write"
+    style={{
+      colorScheme: "light",
+      height: "100%",
+      left: "0px",
+      position: "absolute",
+      top: "0px",
+      width: "100%",
+    }}
+    title="Trace view - Tabbing"
+  ></iframe>
+</div>
+<br />
+Sometimes, duration gaps between entries can be small, but critical to your debugging
+effort. That's why the trace view enables you to zoom into certain areas of the trace,
+either by double clicking on row elements, (which zooms into the row), or by using
+your mouse wheel.
```

Comment:
```suggestion
either by double clicking on row elements (which zooms into the row) or by using
your scroll wheel.
```
I think scroll wheel is a more agnostic term in case users are just on a laptop without a mouse